content/en/docs/concepts/signals/traces.md

```diff
@@ -267,6 +273,19 @@ and an end.
 A Span Event is best used to track the second scenario because it represents a
 meaningful, singular point in time.
 
+#### When to use Span Events or Span Attributes
+
+Span Events contain attributes as well, so you may wonder, when is it appropriate
+to use a Span Event or create a new Attribute on the current span?
+
+It all comes down to if a specific timestamp is meaningful.
+
+For example, if you are tracking an operation with a span and the operation completes,
+you may wish to add data from the operation to your telemetry. If the timestamp in
+which the operation completes is important to know, then a Span Event is an appropriate
+way to model things with telemetry. However, if the timestamp is not meaningful,
+then it's better to attach the data as a Span Attribute instead.
```

Comment:
```suggestion
#### When to use span events versus span attributes

Since span events also contain attributes, the question of when to use events
instead of attributes might not always have an obvious answer. To inform your
decision, consider whether a specific timestamp is meaningful.

For example, when you're tracking an operation with a span and the operation
completes, you might want to add data from the operation to your telemetry.

- If the timestamp in which the operation completes is meaningful or relevant,
attach the data to a span event.
- If the timestamp isn't meaningful, attach the data as a span attribute.
```